tchannel/request.py

```diff
@@ -47,6 +47,16 @@ class Request(object):
         The most useful piece of information here is probably
         ``request.transport.caller_name``, which is the identity of the
         application that created this request.
+
+    :ivar service:
+        Name of the service being called.
+
+    :ivar timeout:
+        Amount of time (in seconds) within which this request is expected to
+        finish.
+
+    :ivar trace:
```

Comment:
Yeah, it does. Currently, using this will override the context's trace info and use the one from the original request.

You think we should just let the default behavior do its thing? That definitely seems more correct because that way the forwarding service itself will show in the trace too.